test/deploy.sh

```diff
@@ -138,8 +138,11 @@ while IFS='' read -r line || [[ -n "$line" ]]; do
   IFS=' ' read ip port mode shardID <<< $line
 	#echo $ip $port $mode
   if [[ "$mode" == "leader" || "$mode" == "validator" ]]; then
-      $DRYRUN $ROOT/bin/harmony -ip $ip -port $port -log_folder $log_folder $DB -min_peers $MIN $HMY_OPT 2>&1 | tee -a $LOG_FILE &
-      sleep 0.5
+     $DRYRUN $ROOT/bin/harmony -ip $ip -port $port -log_folder $log_folder $DB -min_peers $MIN $HMY_OPT 2>&1 | tee -a $LOG_FILE &
+     sleep 0.5
+  fi
+  if [[ "$mode" == "newnode" && "$SYNC" == "true" ]]; then
+     (sleep 35 ; $DRYRUN $ROOT/bin/harmony -ip $ip -port $port -log_folder $log_folder $DB -min_peers $MIN $HMY_OPT 2>&1 | tee -a $LOG_FILE ) &
```

Comment:
If there are multiple newnodes, will all the new nodes sleep 35 seconds and join the network at almost similar time? We want to each of them delay by 35s.

Reply:
I see.  I'll add some logic here.